src/py/flwr/cli/config_utils.py

```diff
@@ -128,3 +129,13 @@ def validate(
         return False, [reason], []
 
     return True, [], []
+
+
+def get_flower_home() -> Path:
+    """Return the Flower home directory based on env variables."""
+    return Path(
+        os.getenv(
+            "FLWR_HOME",
+            f"{os.getenv('XDG_DATA_HOME', os.getenv('HOME'))}/.flwr",
+        )
+    )
```

Comment:
Let's move this function into a new module in `flwr.common`. We also need it in `flower-supernode`.